.github/workflows/build-41.yml

```diff
@@ -2,6 +2,10 @@ name: ublue akmods 41
 on:
   pull_request:
   merge_group:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
```

Comment:
Should this be under pull_request?  None of the others have branches or ignored paths under the MG

Reply:
Thank you. Yes, that should be under pull_request.